apps/vue-showcase/src/app/components/widgets/SideBar.vue

```diff
@@ -0,0 +1,31 @@
+<template>
+  <div class="showcase__app__sidebar">
+    <template v-for="(items, i) in sidebarItems" :key="i">
```

Comment:
Would rename this to singular form, `item`. `items` implies that it is an array.